Assets/Scripts/Player.cs

```diff
@@ -84,107 +40,42 @@ void Update()
             playerVelocity.y = 0f;
         }
 
-        Vector3 move = new Vector3(Input.GetAxis("Horizontal"), 0, Input.GetAxis("Vertical"));
-        float facing = Camera.main.transform.eulerAngles.y;
-        Vector3 turnedMove = Quaternion.Euler( 0, facing, 0) * move;
-        controller.Move(turnedMove * Time.deltaTime * playerSpeed);
-        
+        //Vector3 move = new Vector3(-1 * Input.GetAxis("Vertical"), 0, Input.GetAxis("Horizontal"));
+
+        Vector3 towardsTarget = (target.transform.position - transform.position).normalized;
+        Vector3 tangentToMotionCircle = Vector3.Cross(Vector3.up, towardsTarget); //counterclockwise
+        Vector3 move = towardsTarget * Input.GetAxis("Vertical") + tangentToMotionCircle * Input.GetAxis("Horizontal");
```

Comment:
Not sure if I like this movement in reference to the target, would make movement in freeCam mode very messy and I think code may have to just be brought back to the old stuff anyways

That is unless the movement is the reason the camera was freaking out then we'll just merge this in and figure that part out later

Reply:
I'm going to leave this this way because it was part of my version of the cam fix. I'm scared to take it away. I encourage overriding any of my movement code when the fancy camera guy makes his fancy camera merge

Reply:
Fixed